litmus-portal/frontend/src/views/AnalyticsDashboard/MonitoringDashboardPage/ChaosAccordion/styles.ts

```diff
@@ -0,0 +1,79 @@
+import { makeStyles, withStyles } from '@material-ui/core';
+import MuiAccordion from '@material-ui/core/Accordion';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import MuiAccordionSummary from '@material-ui/core/AccordionSummary';
+
+const useStyles = makeStyles((theme) => ({
+  accordionSummary: {
+    display: 'flex',
+    justifyItems: 'center',
+    background: theme.palette.disabledBackground,
+  },
+
+  accordionDetails: {
+    width: '100%',
+  },
+
+  chaosHelperText: {
+    fontWeight: 500,
+    fontSize: '1rem',
+    color: theme.palette.primary.main,
+  },
+
+  tableDropIcon: {
+    width: '1.75rem',
+    height: '1.75rem',
+    marginTop: theme.spacing(-0.15),
```

Comment:
I'm finding it hard to believe that so many negative top margins are required in the code...I would request you to find them and refactor those places...otherwise, it'll be hard to find later.

Reply:
Removed